5.5/contrib/my-slave.cnf.template

```diff
@@ -3,6 +3,7 @@
 server-id     = ${MYSQL_SERVER_ID}
 log_bin       = ${MYSQL_DATADIR}/mysql-bin.log
 relay-log     = ${MYSQL_DATADIR}/mysql-relay-bin.log
+binlog_do_db  = mysql
 binlog_do_db  = ${MYSQL_DATABASE}
```

Comment:
No overwriting, this is how it's supposed to be. A database name can apparently contain a comma, that's the reason you need two options. See the red warning here: https://dev.mysql.com/doc/refman/5.5/en/replication-options-binary-log.html#option_mysqld_binlog-do-db